### PR TITLE
Minor tweaks to SK cheatsheet translation

### PR DESCRIPTION
**Pull request by eronisko**
## Overview
**TL;DR**
This is a bunch of minor fixes and wording adjustments to Slovak translation of the cheatsheet. 

### Questions
Pinging @marekdlugos (as the main contributor of this translation) for any comments.

### Next Steps
N/A

### Review
Ready for review.

### CODE REVIEW

**Comment by brianamarie** [comment]
Hello @eronisko, and thank you for these contributions! Since nobody on our team is fluent in Slovak, would you be able to help us find another community member to review these changes? Once we have a review from another fluent Slovak speaker, we will merge these changes in. 😊

**Comment by eronisko** [comment]
Hi @marekdlugos, could you please take a look at this?

**Comment by dalmirbortoli** [comment]
vou te pegar vaca puta chupadeira
e picotiar este filho da puta

Em qua, 21 de out de 2020 10:05, Briana Swift <notifications@github.com>
escreveu:

**Comment by eronisko** [comment]
> Once we have a review from another fluent Slovak speaker, we will merge these changes in. 😊

@brianamarie, it looks like the author of the original translation (@marekdlugos) is not responding and it doesn't there are any other Slovak speakers among the contributors to this translation.

Can I ask some of my (Slovak) colleagues to take a look?

**Comment by brianamarie** [comment]
> Can I ask some of my (Slovak) colleagues to take a look?

Yes, absolutely @eronisko :blush:

File: downloads/sk/github-git-cheat-sheet.md
Comment (zuzana1311) [inline downloads/sk/github-git-cheat-sheet.md:5]: "host-ovať"

**Review by zuzana1311 — APPROVED** [review]

**Comment by brianamarie** [comment]
Fabulous, thank you @zuzana1311 and @eronisko! This is ready to merge from my perspective. Are there any further additions or edits you'd like to make before merging?

**Comment by eronisko** [comment]
Thank you @brianamarie and @zuzana1311 🙌 I'm happy for this to be merged.

**Comment by marekdlugos** [comment]
Sorry for the late response! Great work all!

**Pull request merged**
